src/view/Office.vue

```diff
@@ -320,8 +328,13 @@ export default {
 				if (args.success) {
 					this.documentReady()
 				} else {
-					this.error = args.errorMsg
-					this.loading = LOADING_STATE.FAILED
+					if (args.errorType === 'clusterscaling') {
+						this.loadingMsg = args.errorMsg
```

Comment:
Ah the error type is already used, so we could just have something like
```suggestion
						this.loadingMsg = t('richdocuments', 'Cluster is scaling')
```

However it would be good to find a nice message that is not technical as this message is presented to end users not only admins.